### PR TITLE
Add git behind proxy article

### PR DESCRIPTION
**Pull request by alayek**
Added article on how to use git behind proxy server.

A lot of our campers are accessing GitHub in their colleges, or workplace. This would surely serve as a reference for them.

### CODE REVIEW

**Comment by Rafase282** [comment]
# LGTM :+1: 

@theflametrooper if you spot any grammar error let us know.

**Comment by theflametrooper** [comment]
@Rafase282 Looks good! Ready for merging...

File: Git-Behind-Proxy.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Git Behind A Proxy Server
+
+### Use-cases
+
+You might need to modify `git` commands that accesses (to update them and read from) remote repositories if your internet access is through a [proxy server](https://en.wikipedia.org/wiki/Proxy_server).
```
Comment (SeanFCC) [inline Git-Behind-Proxy.md]: * I believe 'accesses' should be changed to 'access'.
* I would do (to update and read from) and take out 'them'.

File: Git-Behind-Proxy.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+It's quite common if you are a student in college, or maybe you are a working professional.
```
Comment (SeanFCC) [inline Git-Behind-Proxy.md]: Maybe reword to something like: 'Proxy servers are common in college and business type environments.'

File: Git-Behind-Proxy.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+You can [locate your proxy settings](http://www.wikihow.com/Change-Proxy-Settings) from your browser's settings panel.
+
+### Using Proxy with Git
+
+Once you have obtained the proxy settings (server URL, port, username and password); you need to configure your git as follows.
```
Comment (SeanFCC) [inline Git-Behind-Proxy.md]: 'as follows:' I believe.

File: Git-Behind-Proxy.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+>- Your system administrator or corporate policy does not allow you to access remote `git` repositories from GitHub, BitBucket etc.
+- The remote repository in question is not in your machine, but it's within the internal network. An instance of GitLab deployed internally at your company is a good example.
+
+### Unset Proxy Settings
+
+Use [this](http://stackoverflow.com/questions/11499805/git-http-proxy-setting) Stack Overflow discussion to unset your proxy settings.
+
+### Resources
+You may use the following for further reading on this
```
Comment (SeanFCC) [inline Git-Behind-Proxy.md]: '..on this:'
